### DISCUSSION TITLE
Can not run Flareact dev

### DISCUSSION THREAD

**Discussion by adembacajglobant** [flareact/flareact#190] [Q&A] [answered]
Hello everyone, I saw this issue reported, but it was closed. I am using a Macbook M1 chip, just pulled a project which is running in the same Macbook M1 chip (my friend), but not on mine. 

I have installed Wrangler, also used wrangler login, added my account_id, but it doesn't start. This is what I got in npm run dev: 

> [worker] Assertion failed: (thread_id_key != 0x7777), function find_thread_id_key, file ../src/coroutine.cc, line 134.
[worker] Error: failed to execute `"/opt/homebrew/bin/node" "/Users/adem.bacaj/Library/Caches/.wrangler/wranglerjs-1.19.4" "--output-file=/var/folders/_v/12khrp1516142h87rfpv2_n40000gs/T/.wranglerjs_outputhMYLD" "--wasm-binding=WASM_MODULE" "--webpack-config=node_modules/flareact/webpack.js"`: exited with signal: 6
[worker] wrangler dev exited with code 1
--> Sending SIGTERM to other processes..
[client] webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development --port 8080 exited with code SIGTERM
[
  {
    command: {
      command: 'wrangler dev',
      name: 'worker',
      prefixColor: '',
      env: [Object]
    },
    index: 0,
    exitCode: 1
  },
  {
    command: {
      command: 'webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development --port 8080',
      name: 'client',
      prefixColor: '',
      env: [Object]
    },
    index: 1,
    exitCode: 'SIGTERM'
  }
]
<img width="1122" alt="Screen Shot 2021-11-01 at 11 18 26 AM" src="https://user-images.githubusercontent.com/86207869/139678505-e78b90c7-0796-41c6-95c0-96dc7b300a34.png">

**Comment by tj-kev** [comment]
I think this is potentially more of a wrangler issue. Don't have access to an M1 to try it out. Are you on the latest version of wrangler?

Found an issue on the wrangler repo regarding issues with M1 which have been fixed in later versions. Doesn't look like the same error message you got though. https://github.com/cloudflare/wrangler/issues/1685 ?

**Comment by adembacajglobant** [comment, marked as answer]
Just fixed the issue. It didn't work with node v16.x.x. 
When I downgraded to node v14.x.x it works good

**Reply by tj-kev** [reply]
Happy you got it sorted. Thanks.

**Reply by shaunm** [reply]
This saved me.

**Reply by tj-kev** [reply]
The canary release should work on Node 16 now. Feel free to give it a try. Any problems let me know.

**Reply by shaunm** [reply]
Okay, will test it. 

> Unrelated question, and I can open an issue if that's a better place, but I have been trying to use the mui package with flareact and have been unsuccessful. I keep getting Error 1101 thrown with the exception: 
> `TypeError: Cannot read properties of null (reading 'registered')`
>
> Might you have any suggestions?


yarn over npm solves a lot of these problems.
